README.md

```diff
@@ -0,0 +1,47 @@
+# Creating a GitHub deployment if PlatformSH has deployed
+
+This action creates a GitHub deployment when PlatformSH has deployed a
+commit.
+
+## Usage
+
+```yaml
+---
+name: "GH Deployment from PSH status"
+
+on:
+  push:
+    branches: [main]
+  status:
+
+permissions:
+  pull-requests: read
+  deployments: write
+  statuses: write
+  contents: read
+
+jobs:
+  deploy:
+    runs-on: ubuntu-24.04
+    if: github.event_name == 'push' || (github.event.context == 'platformsh' && github.event.state != 'pending')
+    steps:
+      - uses: actions/checkout@v4
+      - run: echo "Triggered by ${{ github.event_name }}"
+
+      - uses: reload/vpn-action@main
+        with:
+          server: ${{ vars.RELOAD_HQ_IP }}
+          psk: ${{ secrets.RELOAD_VPN_PSK }}
+          username: ${{ vars.RELOAD_VPN_USERNAME }}
+          password: ${{ secrets.RELOAD_VPN_PASSWORD }}
+          route: 94.23.123.122  # *.webtestbupl.dk
```

Comment:
Måske dette er bedre til en readme? Alternativt kan vi måske slette denne linje - dokumentation til vpn-action må nævne noget om inputs
```suggestion
          route: xx.xx.xxx.xxx
```

Reply:
Ja, i det hele taget er det nok ikke relevant at have VPN med her -- det er jo meget specifikt for enkelte kunder med det behov. Jeg tager den del ud!

Reply:
Jeg har forbedret dokumentationen i det opdaterede commit: 58f7d8ec10ae00e39e452120be2c6dd006ab3e57.